packages/routing-utils/src/types.ts

```diff
@@ -123,5 +123,5 @@ export interface AppendRoutesToPhaseProps {
   /**
    * The phase to append the routes such as `filesystem`.
```

Comment:
Lets updating the JSDoc comment here to mention that `null` represents the phase prior to any `handle` being found.

Reply:
:spiderman-pointing: